core/group/service.go

```diff
@@ -335,6 +337,13 @@ func (s Service) RemoveUsers(ctx context.Context, groupID string, userIDs []stri
 			err = errors.Join(err, currentErr)
 		}
 	}
+
+	if err == nil {
+		audit.GetAuditor(ctx, schema.PlatformOrgID.String()).LogWithAttrs(audit.GroupMemberRemovedEvent, audit.GroupTarget(groupID), map[string]string{
```

Comment:
we should pass org id of the group id instead of platform id here?

Reply:
Done